src/plugins/openmeteo/datasource.py

```
@@ -0,0 +1,333 @@
+    async def get_measurements(
+        self,
+        sensor: Sensor,
+        start_date: Optional[datetime] = None,
+        end_date: Optional[datetime] = None,
+        limit: Optional[int] = None,
+        **filters: Any
+    ) -> AsyncIterator[List[Measurement]]:
+        try:
+            session = await self._get_session()
+            
+            start = start_date or datetime.now() - timedelta(days=7)
```

Ruff (0.12.2)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)